src/posts/blog/7-budget/budget.ts

```diff
@@ -0,0 +1,91 @@
+
+We encountered questions like:
+* “What’s riskier: a 3-hour unmasked. outdoor hang? Or a 30-minute indoor, masked hang? 🤷‍♂️”
+* “Sam really wants to see their bodyworker. I really wanted to see my partner. But was I ok being in indirect contact with that bodyworker for Sam's sake? Did Sam think my happiness from seeing my partner was worth the risk they'd experience?”
+
+Then we disocvered the microCOVID spreadsheet. I realized it had the potential to completely change how we had these discussions. If we could quantify and track how much risk each of us were taking, we could allocate everyone a small but even share of risk that they could use to make their own decisions, without consulting the house every time.
+
+I'll say this: since we replaced our consensus approach with a microCOVID budget approach, COVID-related relationship stress in the house has gone down dramatically. We can now make choices on our own about what we do that’s within our individual risk budget! Once the system is set up and everyone knows how to use it, we don’t need to have many (if any) group meeting time to talk about our COVID agreements.
+
+**Having this new approach has saved us SO much energy. We got our lives back.**  We aren’t constantly talking about COVID decisions every week, and we have flexibility to take certain actions that are important to us.
+
+I realized that, although I personally love spreadsheets, many people don’t. I wanted to find a way for everyone to be able to get their lives back the way my house did. That’s why I’ve been working with the microCOVID team for the last few months to upgrade the risk tracker so it is cleaner, clearer, and easier to use. I want anyone to be able to pick it up and start using it for their house/pod. We hope you find it useful for your situation!
+
+
+----------
+## What’s new in Version 2?
+
+Version 1 of the Risk Tracker was built primarily for [individual risk budgeting](/paper/2-riskiness#choosing-your-risk-budget) that wasn’t possible on the [website calculator](/).
+
+**Version 2 is built with households/pod risk budgeting in mind** (and it’s an upgrade for individual budgeting as well!).
+
+Here’s what’s you’ll find in the latest version:
+
+
+- **Household/pod risk budget tracking** to empower individuals within your group to have more flexibility to make decisions while keeping everyone’s risk within an agreed-upon risk threshold.
+- A new [**quickstart guide**](/tracker/quickstart) to orient you to the spreadsheet.
+- [**Step-by-step instructions**](/tracker/basics) to help answer many common questions about the spreadsheet and how to use it in specific scenarios.
+- [**Custom person risk modeling**](/tracker/basics#using-the-custom-people-sheet-to-model-risk-more-precisely), so you can figure out just how risky that friend or co-worker is to see.
+- **Updated mask options** and risk multipliers that match the Calculator. (See our recent blog posts on [upgrading your mask](/blog/masks).)
+
```

Comment:
Did we decide not to mention the new decay function?

Reply:
Fine either way, just wanted to check that this was a conscious decision. It feels like an important thing to houses the are in microCOVID networks.

Reply:
I'll add it.